R/bibtex.R

```diff
@@ -1,6 +1,17 @@
 # backport of simplified trimws() (introduced in R-3.3.0)
-trimws <- function(x) {
-  sub("^[[:space:]]+", "", sub("[[:space:]]$", "", x))
+# See https://github.com/wch/r-source/blob/trunk/src/library/base/R/strwrap.R
+trimws <- function(x, which = c("both", "left", "right"),
```

Comment:
Updated version of the backport

Reply:
Should we consider a dependency on the [`{backports}`](https://github.com/r-lib/backports) package to automatically get up-to-speed? Though, as it stands now, we only have a dependency on one base package: `utils`. 

That said, we probably should bump the minimum version to at least R 3.6.* as we are not really testing if it works on R 3.0. Something to ponder for later on.

Reply:
Good idea, I just added the `backports`, but agree on bumping the minimal R version as well